Cybirical/django-bulk-load
# Adding support for the Django5+ GeneratedField

## Summary
Django 5 introduced the ability to have GeneratedFields on a model. These models are not allowed to be modified directly and are causing issues on bulk_insert and bulk_upsert. 

## Test Plan
##### How can reviewers test your change manually?
1. Setup a model with a GenerateField
2. Attempt to perform a bulk_insert or bulk_upsert using the library
3. Before applying this upgrade you should see it fail with the issue: `raise AttributeError( AttributeError: Cannot read a generated field from an unsaved model.`

##### What automated tests did you add? If none, please state why.
Automated tests have been added exercising the insert, update and upsert methods.
